indico/htdocs/js/indico/jquery/global.js

```diff
@@ -166,8 +166,7 @@ $(document).ready(function() {
     });
 
     $('.js-dropdown').each(function() {
-        this.id = this.id || uniqueId();
```

Comment:
Do you know if we actually use a dropdown inside a dropdown somewhere? I couldn't find one.